packages/react/src/publication/useExplorePublications.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  PublicationSortCriteria,
+  PublicationTypes,
+  useExplorePublicationsQuery,
+} from '@lens-protocol/api-bindings';
+
+import { PaginatedArgs, PaginatedReadResult, usePaginatedReadResult } from '../helpers';
+import { useSharedDependencies } from '../shared';
+import { DEFAULT_PAGINATED_QUERY_LIMIT, DEFAULT_PUBLICATION_SORT_CRITERIA } from '../utils';
+import { Publication } from './types';
+
+type UseExplorePublicationsArgs = PaginatedArgs<{
+  observerId?: string;
+  sortCriteria?: PublicationSortCriteria;
+  timestamp?: number;
+  noRandomize?: boolean;
+  publicationTypes?: Array<PublicationTypes>;
+  excludeProfileIds?: Array<string>;
+}>;
+
+export function useExplorePublications({
+  observerId,
+  limit = DEFAULT_PAGINATED_QUERY_LIMIT,
+  sortCriteria = DEFAULT_PUBLICATION_SORT_CRITERIA,
+  timestamp,
+  noRandomize,
+  publicationTypes,
+  excludeProfileIds,
+}: UseExplorePublicationsArgs = {}): PaginatedReadResult<Array<Publication>> {
```

Comment:
Can we please clarify what is the mindset with making these specific set of parameters available?

Not sure some of these are intelligible (e.g. `noRandomize`) as they are.

Reply:
I feel like we should give the user the ability to choose if they want to use a parameter or not, even if it's an obscure one. Putting that matter aside i think `noRandomize` and `customFilters` (added in my last commits) are the only ones that are not self-explanatory. `customFilters` is being used in some big UIs like Lenster so I think it should be added.

Reply:
I opted for deleting those parameters, we can think in the future if it's worth adding